models/article-models.js

```diff
@@ -59,5 +60,3 @@ exports.checkArticleExists = (article_id) => {
       }
     });
 };
-
```

Comment:
I cant see the controller on this PR, but in your model, you don't take the comments off the rows key - which I recommend doing in your controller